projects/arlas-components/src/lib/components/mapgl/model/map.type.ts

```diff
@@ -340,11 +210,4 @@ export interface MapOverride {
   once<T extends keyof unknown>(type: T, listener: (ev: unknown) => void): this;
   once(type: string, listener: (ev: any) => void): this;
 
-  off<T extends keyof unknown>(
-    type: T,
-    layer: string,
-    listener: (ev: unknown) => void,
-  ): this;
-  off<T extends keyof unknown>(type: T, listener: (ev: unknown) => void): this;
-  off(type: string, listener: (ev: any) => void): this;
```

Comment:
Event if its not use i think is useful to keep theses method. Cause if we keep on, once method, we should implement a way of remove the event we created.